packages/frontend-utils/src/api-util/api-util.ts

```diff
@@ -40,10 +39,6 @@ export const createSentryHeaders = (): {
   };
 };
 
-export const createFeatureFlagHeaders = () => ({
```

Comment:
Should we not keep this for future feature flags

Reply:
This has been added specifically for this case to switch between squidex and contentful https://github.com/yldio/asap-hub/pull/2770/files#diff-5a88dd4b45d008f2791dc6fa0772cde8df72c13c63df9adedd19fd5f3e949bbb. I think we could just remove this one and if we ever need another feature flag maybe the person could implement a new header or way to switch a feature. What do you think?